pkg/storage/watch/monitor.go

```diff
@@ -0,0 +1,27 @@
+package watch
+
+import "sync"
+
+type Monitor struct {
```

Comment:
godoc here
does this have to be in this package or could it be under e.g. 'util/monitor`? it doesn't seem watch-related really

Reply:
Yes I was thinking about moving it there, good catch!